### ISSUE TITLE
bought a subscription but didn't log in.

### ISSUE THREAD

**Issue by yulantarwiyahtuladawiyah** [NopeCHALLC/nopecha-extension#80]
Hi,
I just purchased a subscription for $39.99 paid via Paypal. My Paypal charged me for the amount but I don't see any subscription in my account and it says it will appear in a few minutes. But I don't see it even after 30 minutes.

**Comment by yulantarwiyahtuladawiyah** [comment]
please respond, how to solve it. you are really harming me. 40$ is not a small amount for me

**Comment by WindFlarer** [comment]
yes I was also scammed out of 5 dollars.

**Comment by jung-min-seo** [comment]
here, too

**Comment by DzT13** [comment]
if anyone purchase with paypal, you can ask refund in paypal.

**Comment by yulantarwiyahtuladawiyah** [comment]
Yes, I paid using Paypal. Can it be refunded if I paid using Paypal?

**Comment by DzT13** [comment]
yes you can, search on yt for how to refund that if you don't know to refund. Don't forget to give paypal evidence that Nopecha didn't respond. Screen shoot anything what you have done to contact the seller. and give details how you face this problem.

**Comment by darlopvil** [comment]
i was gonna make an issue about this. This past Monday i got my new sub renovation and event today (Friday) i haven't seen the new sub on my panel. This isn't funny. Discord server also doesn't help. You cannot ask for help/support but asking the bot for free subscriptions?! wtf
Gonna do the refund thing on PayPal too. Hope these guys will see anything...

**Comment by lee23651** [comment]
Please use our Email support@nopecha.com for support, although these cases were caused by payment processing delays which have been fixed for a long time.
